CHANGELOG.md

```diff
@@ -1,23 +1,64 @@
-x.y.z Release notes (yyyy-MM-dd)
+10.0.0-beta.3 Release notes (yyyy-MM-dd)
+
 =============================================================
+
 ### Enhancements
-* None.
+
+* The following classes & aliases have been renamed:
+
+`RLMSyncUser` -> `RLMUser`
+`SyncUser` -> `User`
+
+`RLMAppCredential` -> `RLMCredential`
+`AppCredential` -> `Credential`
+
+`RealmApp` -> `App`
+
+`RLMUserAPIKeyProviderClient` -> `RLMAPIKeyAuth`
+`UserAPIKeyProviderClient` -> `APIKeyAuth`
+
+`RLMUsernamePasswordProviderClient` -> `RLMEmailPasswordAuth`
+`UsernamePasswordProviderClient` -> `EmailPasswordAuth`
+
+* The following functionality has also moved to the User
+
+`[RLMApp callFunctionNamed:]` -> `[RLMUser callFunctionNamed:]`
+`App.functions` -> `User.functions`
+
+`[RLMApp mongoClientWithServiceName:]` - > `[RLMUser mongoClientWithServiceName:]`
+`App.mongoClient(serviceName)` -> `User.mongoClient(serviceName)`
+
+`[RLMApp userAPIKeyProviderClient]` -> `[RLMUser apiKeyAuth]`
+`App.userAPIKeyProviderClient` -> `App.apiKeyAuth()`
+
+`[RLMApp logOut:]` -> `[RLMUser logOut]`
+`App.logOut(user)` -> `User.logOut()`
+
+`[RLMApp removeUser:]` -> `[RLMUser remove]`
+`App.remove(user)` -> `User.remove()`
+
+`[RLMApp linkUser:credentials:]` -> `[RLMUser linkWithCredentials:]`
+`App.linkUser(user, credentials)` -> `User.link(credentials)`
+  
+* Other enhancements
+
+-  `refreshCustomData()` on User now returns the custom data on success.
```

Comment:
refreshCustomData returns void and passes the custom data to the callback on success. This also is a breaking change, not an enhancement.